### PR TITLE
auto OCP-25765

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei  please help to review, thanks!

### CODE REVIEW

File: lib/rules/web/admin_console/4.3/common_ui_elements.xyaml [hunk 1]
```diff
@@ -233,9 +233,16 @@ click_secondary_menu:
   - selector:
       xpath: //a[@data-component='pf-nav-expandable' and text()='<primary_menu>']
     op: click
-  - selector:
+  - selector: &secondary_menu
       xpath: //a[@class='pf-c-nav__link' and text()='<secondary_menu>']
     op: click
+check_secondary_menu:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: Looks like it's not used, so can we just remove it?

Comment (SSshahan) [reply lib/rules/web/admin_console/4.3/common_ui_elements.xyaml]: https://github.com/openshift/cucushift/blob/d736063a90bb9100e221f5567f2df12c2f90c910/web/admin-console/user-management.feature#L43

**Comment by SSshahan** [comment]
@pruan-rht  could you help to merge? thanks

**Comment by SSshahan** [comment]
@pruan-rht could you help to merge it? it will failed in ci run since the case has no relevant rules. Thanks!

**Comment by SSshahan** [comment]
@pruan-rht  conflict issue has solved, could you help to merge? thanks!

**Comment by SSshahan** [comment]
@pruan-rht  could you help to merge it?

**Pull request merged**